src/resolvelib/resolvers.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, provider, reporter):
         self._p = provider
         self._r = reporter
         self._states = []
+        self._failure_casues = []
```

Comment:
This looks like a typo - the other changes use `_failure_causes`

Reply:
It's not a typo, not sure what you mean though

Reply:
This is the only place you use the spelling "casues" (with the "u" before the "e") rather than "causes" ("u" first")...

Reply:
Oh I see, yes it's a typo, let me fix that!